deltachat-ffi/deltachat.h

```diff
@@ -5178,6 +5178,55 @@ int64_t          dc_lot_get_timestamp     (const dc_lot_t* lot);
  */
 
 
+
+/**
+ * @class dc_json_api_instance_t
+ *
+ * Opaque object for using the json rpc api from the cffi bindings.
+ */
+
+/**
+ * Create the jsonrpc instance that is used to call the jsonrpc.
+ *
+ * @memberof dc_accounts_t
+ * @param account_manager The accounts object as created by dc_accounts_new().
+ * @return Returns the jsonrpc instance, NULL on errors.
+ *     Must be freed using dc_json_api_unref() after usage.
+ *
+ */
+dc_json_api_instance_t* dc_get_json_api(dc_accounts_t* account_manager);
```

Comment:
For consistency, I'd propose to either stick to "JSON RPC" or "JSON API" in naming, but not a mixture of both. I think JSON RPC makes more sense, as that's what it is. In that spirit, I'd propose to rename the FFI methods. I'll include suggestions in the review comments.